tests/test_translator.py

```
@@ -97,6 +100,49 @@ def test_translate_text(translator_instance):
     )
     assert "response" in translator_instance.translation_log["translation"]
     assert "usage" in translator_instance.translation_log["translation"]
+    assert "context" in translator_instance.translation_log["translation"]
+    assert translator_instance.translation_log["translation"]["context"] == ""
+
+
+def test_translate_text_with_context(translator_instance):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_translate_text_with_context`

Add return type annotation: `None`

(ANN201)